### PR TITLE
integrate assets cdn

### PR DESCRIPTION
**Pull request by cfaur09**
## Reasoning
- refactored asset retrieval process to eliminate dependency on reading from mx-assets repo
  
## Proposed Changes
- integrate assets-cdn service to read all the informations from assets ( `accounts`, `tokens`, `identities` )

## How to test
- check if assets are retrieved correctly from assets-cdn

### CODE REVIEW

File: src/common/assets/assets.service.ts [hunk 1]
```diff
@@ -1,190 +1,80 @@
...
-
     return allCollectionRanks[identifier];
   }
 
   async getAllCollectionRanks(): Promise<{ [key: string]: NftRank[] }> {
     return await this.cachingService.getOrSet(
       CacheInfo.CollectionRanks.key,
       async () => await this.getAllCollectionRanksRaw(),
-      CacheInfo.CollectionRanks.ttl,
+      CacheInfo.CollectionRanks.ttl
```
Comment (bogdan-rosianu) [inline src/common/assets/assets.service.ts:58]: add `,` back

File: src/common/assets/assets.service.ts [hunk 2]
```diff
@@ -1,190 +1,80 @@
...
-        const tokenAssetsPath = path.join(assetsPath, identifier);
-        const ranks = this.readTokenRanks(tokenAssetsPath);
-        if (ranks) {
-          result[identifier] = ranks;
-        }
+
+    for (const asset of assets) {
+      if (asset.ranks && asset.ranks.length > 0) {
+        result[asset.identifier] = asset.ranks.map((rank: any) => {
+          const nftRank = new NftRank();
```
Comment (bogdan-rosianu) [inline src/common/assets/assets.service.ts:73]: can we add a partial constructor for NftRank?

File: src/common/assets/assets.service.ts [hunk 3]
```diff
@@ -195,41 +85,23 @@ export class AssetsService {
     return await this.cachingService.getOrSet(
       CacheInfo.AccountAssets.key,
       async () => await this.getAllAccountAssetsRaw(),
-      CacheInfo.AccountAssets.ttl,
+      CacheInfo.AccountAssets.ttl
```
Comment (bogdan-rosianu) [inline src/common/assets/assets.service.ts:88]: add `,`

Comment (cfaur09) [reply src/common/assets/assets.service.ts:88]: done

File: src/common/assets/assets.service.ts (hunk 1 above)
Comment (cfaur09) [reply src/common/assets/assets.service.ts:58]: done

File: src/common/assets/assets.service.ts [hunk 4]
```diff
@@ -1,190 +1,80 @@
...
-
-    return name;
-  }
-
   async getAllTokenAssets(): Promise<{ [key: string]: TokenAssets }> {
     return await this.cachingService.getOrSet(
       CacheInfo.TokenAssets.key,
       async () => await this.getAllTokenAssetsRaw(),
-      CacheInfo.TokenAssets.ttl,
+      CacheInfo.TokenAssets.ttl
```
[the remark this answers is not in the document]
Comment (cfaur09) [reply src/common/assets/assets.service.ts:30]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by gabrielmatei — APPROVED** [review]

**Review by gabrielmatei — DISMISSED** [review]

**Review by dragos-rebegea — DISMISSED** [review]

**Review by gabrielmatei — APPROVED** [review]

**Pull request merged**